src/service/notification_service/bill_action_event_handler.rs

```diff
@@ -62,12 +62,13 @@ impl NotificationHandlerApi for BillActionEventHandler {
         true
     }
 
-    async fn handle_event(&self, event: EventEnvelope) -> Result<()> {
+    async fn handle_event(&self, event: EventEnvelope, identity: &str) -> Result<()> {
```

Comment:
if this is a node_id, I think `node_id` would be better, since it implies either personal, or company identity.
if it's a specific personal identity node_id, then maybe identity_node_id :thinking: